README.md

```diff
@@ -69,6 +69,8 @@ If using Mac OS X, you can get the ZeroMQ dependencies using Homebrew:
 $ brew install libsodium zeromq czmq
 ```
 
+Or you can use Nix mentioned below, which also manages system dependencies.
```

Comment:
I’d prefer to keep this section simple and don’t burden contributors with choices here.

Like I said, people who use nix should know what they are doing and they will just run the command below.